tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py

```diff
@@ -0,0 +1,88 @@
+from ruamel import yaml
+
+import great_expectations as ge
+from great_expectations.core.batch import BatchRequest, RuntimeBatchRequest
+
+CONNECTION_STRING = "mysql+pymysql://root@localhost/test_ci"
+# CONNECTION_STRING = "mysql+pymysql://root@127.0.0.1:8889/test_ci"  # TODO REMOVE THIS and UPDATE LINE NUMBERS
+
```

Comment:
I think you intended to change this..? :)

Reply:
aaaack yes thank you.

Reply:
Fixed